src/clr-angular/forms/combobox/options.spec.ts

```diff
@@ -28,19 +28,19 @@ class TestComponent {}
             Test
         </clr-options>
     `,
-  providers: [IfOpenService],
+  providers: [ClrPopoverToggleService],
 })
 class TestComponentWithError {}
 
 export default function(): void {
   describe('Select Options Menu Component', function() {
     let context: TestContext<ClrOptions, TestComponent>;
-    let ifOpenService: IfOpenService;
+    let ifOpenService: ClrPopoverToggleService;
```

Comment:
nit: I would like to keep the naming consistent. Can this be updated to `toggleService` or `popoverToggleService`?

Reply:
Thanks! Just missed it.